.php_cs.dist

```diff
@@ -5,28 +5,74 @@ $config = PhpCsFixer\Config::create()
     ->setRules([
         '@PSR2' => true,
         'array_syntax' => ['syntax' => 'short'],
+        'blank_line_after_opening_tag' => true,
+        'compact_nullable_typehint' => true,
+        'concat_space' => ['spacing' => 'one'],
+        'declare_equal_normalize' => ['space' => 'none'],
         'declare_strict_types' => false,
+        'dir_constant' => true,
+        'final_static_access' => true,
+        'fully_qualified_strict_types' => true,
         'function_to_constant' => ['functions' => ['php_sapi_name']],
```

Comment:
do we want to add things like `php_version` in here?

Reply:
used true instead.